Sources/Rosetta/PlayMode/Conditions/SelfCondition.cpp

```diff
@@ -420,6 +420,18 @@ SelfCondition SelfCondition::IsOverloadCard()
     });
 }
 
+SelfCondition SelfCondition::IsComboCard()
+{
+    return SelfCondition([=](Playable* playable) -> bool {
+        if (playable->GetGameTag(GameTag::COMBO) >= 1)
```

Comment:
The return value of method `GetGameTag(GameTag::COMBO)` is 0 or 1.
So, I suggest change the code to`return playable->GetGameTag(GameTag::COMBO) == 1;`.

Reply:
Good. Sorry for having not checked it.

Reply:
Could you change your code by applying it?

Reply:
I do it :D